src/Symfony/Component/Workflow/Tests/fixtures/puml/arrow/complex-state-machine-marking.puml

```diff
@@ -15,7 +15,7 @@ state "b"
 state "c" <<marked>>
 state "d"
 "a" --> "b": "t1"
-"d" -[#Red]-> "b": "<font color=Grey>My custom transition label 3</font>"
+"d" -[#Red]-> "b": "<font color=Grey>My custom transition</font>\n<font color=Grey>label 3</font>"
```

Comment:
I fail to see why the previous code does not work. In this [answer](https://github.com/symfony/symfony/issues/49311#issuecomment-1441122228), the code is: 
```
"created" --> "started": "<font color=red>start\nhello</font>"
```

but with your previous PR, it should be:
```
"created" --> "started": "<font color=red>start hello</font>"
```

right?

Reply:
My previous PR removed the new line, which with hindsight is not the best way to handle new lines.

PUML can handle new lines as long as we use the `\n` string instead of an actual line, but it breaks the colour added through `<font>`. Here is the result of this first example:

> ![image](https://user-images.githubusercontent.com/2071331/221024192-451a5578-b8d2-40bc-a113-27a4bd1d94a7.png)

So I suggest to do it properly this time.

Reply:
I don't understand how a **space** can be rendered by a new line 🤔 
I mean, there is no bug anymore, but there is a better way to enhance the current situation, right?

Reply:
In the previous PR, I replaced the new lines with spaces, with this PR I render `\n` and PUML will render it as a new line.: https://github.com/symfony/symfony/pull/49516/files#diff-ed45286ac5b7a302737e048e779c9bf08f3e2c15d4206961ed1ef13a3d1a7dc3R220

In summary:

1. before the first PR: new lines broke the rendering of PUML dumps
2. with the previous PR: new lines were removed and the output was valid but colours were broken
3. with this PR: new lines will be visible and colours will be displayed properly

Reply:
> * with the previous PR: new lines were removed and the output was valid but colours were broken

I still don't understand why 😂

Reply:
Crap, what I wrote was wrong, sorry. :bow: 

I mean:

1. before the first PR: new lines broke the rendering of PUML dumps, it crashed the PUML parser
2. after the previous PR #49272 (= today on the 5.4 branch): new lines are removed, the PUML parser is happy, the output is valid and colours are applied **BUT** it misses the point: PUML support new lines, why not use them?
3. with this PR: we leverage the rendering of new lines through PUML, by using `\n` the PNG, etc. output will render new lines as is and colours will be displayed on each line (see the second image in the description)